workflows/raredisease.nf

```diff
@@ -155,6 +158,18 @@ workflow RAREDISEASE {
     )
     ch_versions = ch_versions.mix(CALL_STRUCTURAL_VARIANTS.out.versions)
 
+    ch_sv_annotate = Channel.empty()
+    if (params.svdb_query_dbs) {
```

Comment:
Think about whether the switch should be on file or subworkflow/module level.

Reply:
Good point! I have now moved the switch to the subworkflow level 😄